.github/workflows/release.yml

```diff
@@ -53,6 +53,12 @@ jobs:
         GH_USER_EMAIL: developer-toolkit-team@newrelic.com
         GH_USER_NAME: 'New Relic Developer Toolkit Bot'
 
+    - name: Upload chocolatey package
+      shell: bash
+      run: make chocolatey-publish
```

Comment:
Could this not be part of `make release-publish` ?

Reply:
Maybe yes.  We require that the windows build exists there first before we try to curl it down to package it, so this seemed like a safe place to ensure the rest of the build had been done.